CONTRIBUTING.md

```diff
@@ -17,6 +17,107 @@ They prefer those direct inquiries go through direct messages on gitter or by me
 You can see the list of [maintainers](#maintainers) below.
 
 
+## What to Work On
+
+There is *a lot* of work to do on this project, no matter your area of expertise or skill level.
+
+If you are a beginner there are several [Github issues](https://github.com/cocotb/cocotb/issues) marked [`good first issue`](https://github.com/cocotb/cocotb/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) you can look at.
+There are also a number of thins you can work on that *aren't* in Github issues;
+including the following
+
+* Documentation improvements
+* Managing Github issues and the gitter
+* Test and coverage improvement
+
+
+#### Documentation
+
+Cocotb's documentation is always open to improvements.
+Improving documentation will help users better understand and use cocotb;
+and may decrease the number of questions the gitter and Github issue page see.
+Updating documentation requires the following.
+
+* A good grasp on English
+* An understanding of [Markdown](https://www.markdownguide.org/)
+* [reStructuredText](https://docutils.sourceforge.io/rst.html)
+* [Sphinx documentation generator](https://www.sphinx-doc.org/en/master/)
+
+Some documentation should be located in the official documentation on [ReadTheDocs](https://docs.cocotb.org/en/stable/),
+while the rest belongs on the [Wiki](https://github.com/cocotb/cocotb/wiki).
+There are several ways to improve the documentation.
+
+* Better documenting core functionality (RTD)
+* Documenting common "gotchas" (RTD)
+* Documenting difficult and niche use cases (Wiki)
+* Documenting common design patterns (Wiki)
+* Documenting internal components (Wiki)
+
+Documentation improvements typically require no maintainer buy-in.
+You can simply work on the documentation and open a pull request.
+Please see the guidelines on submitting [pull requests](#patch-requirements).
+Documentation on the Wiki does not require a pull request, and any user with a Github account can contribute to it.
+Please be responsible with that power.
+
+
+#### Project Management
+
+The cocotb project is [fairly popular](https://larsasplund.github.io/github-facts/verification-practices.html#frameworks) and the
+[gitter](https://gitter.im/cocotb/Lobby) and
+[Github issues](https://github.com/cocotb/cocotb) page recieve a fair amount of traffic;
+which is only expected to increase.
+People are needed to categorize issues and pull requests, and respond to questions.
+Working this task is the quickest way to learn how cocotb works and become familiar with how cocotb works.
```

Comment:
The words before and after the "and" are saying essentially the same thing, no?

Reply:
Whoops. Partial refactor.